Examples/Io/HepMC3/include/ActsExamples/Io/HepMC3/HepMC3Reader.hpp

```diff
@@ -1,21 +1,31 @@
 // This file is part of the Acts project.
 //
-// Copyright (C) 2018 CERN for the benefit of the Acts project
+// Copyright (C) 2018-2020 CERN for the benefit of the Acts project
 //
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 #pragma once
 
+#include "ActsExamples/Framework/IReader.hpp"
+#include <Acts/Utilities/Logger.hpp>
 #include <HepMC3/GenEvent.h>
 #include <HepMC3/ReaderAscii.h>
 
 namespace ActsExamples {
 
 /// HepMC3 event reader.
-struct HepMC3ReaderAscii {
- public:
+struct HepMC3ReaderAscii : public IReader {
+  struct Config {
+    // The input directory
+    std::string inputDir = "";
```

Comment:
```suggestion
    std::string inputDir;
```
Nitpicking: default initialization already provides an empty string.

Reply:
done